cluster/mesos/cluster.go

```diff
@@ -177,6 +177,15 @@ func (c *Cluster) UnregisterEventHandler(h cluster.EventHandler) {
 	c.eventHandlers.UnregisterEventHandler(h)
 }
 
+// StartContainer starts a container
+func (c *Cluster) StartContainer(container *cluster.Container) error {
+	// if the container was started less than a second ago, do not start it
+	if time.Now().Unix()-container.Created > 1 {
```

Comment:
And also if somehow there is a delay more than 1 second of the second request we will still see double start right?